scrapypuppeteer/actions.py

```diff
@@ -159,6 +159,9 @@ def payload(self):
             'navigationOptions': self.navigation_options
         }
 
+    def __bool__(self):
+        return not not self.selector
+
```

Comment:
We cannot disallow empty selectors as they are used to show that there is no 'submit recaptcha' button. But I agree with absurdity of Bool-ish actions.
We have two options:
1.) Just check if submitting.selector is not an empty string
2.) In case of empty string provided transform it into None
Now I emplement first option. Notice me if you thought of something better